tests/src/test/scala/org/apache/openwhisk/core/containerpool/test/ContainerPoolTests.scala

```diff
@@ -67,6 +68,9 @@ class ContainerPoolTests
   // the values is done properly.
   val exec = CodeExecAsString(RuntimeManifest("actionKind", ImageName("testImage")), "testCode", None)
   val memoryLimit = 256.MB
+  val ttl = FiniteDuration(2, TimeUnit.SECONDS)
```

Comment:
I would make ttl shorter - so that test runs don't take longer than necessary. Maybe 500.millis?

Reply:
Great! updated accordingly